Assets/UGSSamples/Plugins/Unity.Service.Samples.asmdef

```diff
@@ -1,7 +1,10 @@
 {
     "name": "Unity.Service.Samples",
     "rootNamespace": "",
-    "references": [],
+    "references": [
```

Comment:
how come we did not have them linked before?

Reply:
I don't think we needed them before, none of the code referenced them?